webpack.config.cjs

```diff
@@ -44,7 +44,8 @@ module.exports = () => {
         mode,
         context: path.resolve(__dirname),
         resolve: {
-            modules: [path.resolve(__dirname, 'src'), 'node_modules'],
+            exportsFields: ['itowns_exports', 'exports'],
+            modules: [path.resolve(__dirname, 'src'), path.resolve(__dirname, './packages/Geodesy/src'), 'node_modules'],
```

Comment:
Shall this not be `babel` which should resolve the files dependencies (i.e. in `include`)?

Reply:
I think that eslint seems to need the paths to resolve